templates/python/example.py

```diff
@@ -1,20 +1,23 @@
-#
-#  Copyright (c) 2023-2024 - Restate Software, Inc., Restate GmbH
-#
-#  This file is part of the Restate examples,
-#  which is released under the MIT license.
-#
-#  You can find a copy of the license in file LICENSE in the root
-#  directory of this repository or package, or at
-#  https://github.com/restatedev/sdk-typescript/blob/main/LICENSE
+import uuid
+from datetime import timedelta
 
 from restate import Service, Context
 import restate
 
-greeter = Service("greeter")
+from utils import send_notification, send_reminder
+
+greeter = Service("Greeter")
+
 
 @greeter.handler()
 async def greet(ctx: Context, name: str) -> str:
-    return f"Hello {name}!"
+    # Durably execute a set of steps; resilient against failures
+    greeting_id = await ctx.run("generate UUID", lambda: str(uuid.uuid4()))
```

Comment:
Opened an issue for this https://github.com/restatedev/sdk-python/issues/30

Reply:
I'll change it once it's there